0006/README.md

```diff
@@ -222,69 +247,43 @@ Pages.
 -   The usage of Git submodule is non-standard, and it leaves some versioned
     directories in the `main` branch.
 
-*What is changing and code snapshot*
-
-The following is a snapshot of the scripts for building and deploying the
-website.
+*What is changing*
 
--   To serve the website locally,
+-   To serve the website locally, step 2 in
     [/docs/README.md](https://github.com/slsa-framework/slsa/blob/main/docs/README.md)
-    is replaced with running `build.sh` or similar logic.
--   The Github Workflow uses the same shell script to prepare the build
-    environment.
+    is updated to be:
+
+    ```bash
+    git clone --recurse-submodules https://github.com/chtiangg/slsa.git
+    git submodule update --remote
+    cd slsa/docs
+    ```
+-   A new Github Workflow script builds and deploys the website. The following
+    is a cod snapshot.
```

Comment:
nit: what is a "cod snapshot"?

Reply:
Will also update this after our meeting.

Reply:
Done.